anda/langs/nim/dive/umdive.spec

```diff
@@ -0,0 +1,29 @@
+Name:           umdive
```

Comment:
Is there a reason why this is called umdive instead of dive?

Reply:
https://pkgs.org/download/dive

Just want to avoid naming collisions at all cost